src/components/Table/types.ts

```diff
@@ -100,6 +104,29 @@ export type RowsConfig = {
   setExpanded?: OnChangeFn<ExpandedState>;
 };
 
+/** Pagination */
+
+export type PaginationConfig = {
+  /** The current page displayed */
+  page: number;
+  /** Total pages count */
+  totalPages: number;
+  /** OnChange callback for next/prev buttons */
+  onChange: (page: number) => void;
+  /** Whether go-to-first-page and go-to-last-page buttons are visible */
+  isEnhancedPaginationVisible?: boolean;
+  /** Manually disable next page */
+  isNextPageDisabled?: boolean;
+  /** Manually disable previous page */
+  isPrevPageDisabled?: boolean;
+  /** Show items per page list options */
+  itemsPerPageOptions?: SelectOption[];
```

Comment:
selectOption has many things

https://designlab-pr-815.orfium-dev.com/?path=/docs/updated-components-fields-select--overview

Do we need them all ?

Reply:
since we are using Select component for itemsPerPage, shouldn't we use the necessary component types?

Reply:
we should but select comes with a lot of other properties. We can omit to secure the functionality as user doesn't have access to select of pagination

Reply:
instead of `SelectOption` we could use `Omit<SelectOptionValues, 'iconProps'>`, 
which equals `{id: string | number, value: string}`. 

something like this maybe?